### PR TITLE
Now request params is a plain string

### PR DESCRIPTION
**Pull request by thedae**

### CODE REVIEW

File: lib/carto/common/rack_logger_middleware.rb [hunk 1]
```diff
@@ -37,6 +38,8 @@ def deep_obfuscate_values(hash)
             deep_obfuscate_values(value)
           elsif value.is_a?(String)
             hash[key] = LOGGABLE_PARAMS.include?(key.to_s) ? value : ('*' * value.length)
+          elsif value.is_a?(Array)
+            hash[key] = value.join(',')
```
Comment (amiedes) [inline lib/carto/common/rack_logger_middleware.rb:42]: If the value was something like `["personal_info_1", "personal_info_2"]` would't we be sending it to the logs in plain as `personal_info_1, personal_info_2`?

I think we should obfuscate it also if it's not a `LOGGABLE_PARAMS `

**Review by amiedes — CHANGES_REQUESTED** [review]
I've just left a minor comment 🙂

File: lib/carto/common/rack_logger_middleware.rb (hunk 1 above)
Comment (thedae) [reply lib/carto/common/rack_logger_middleware.rb:42]: Before these changes the key-pair of an array was something like:
```
key1=value1&key2="Instance of Array"
```
which looked weird, that's why I added these lines.

Anyway, you're right, this value should be obfuscated, I'm on it!

File: lib/carto/common/rack_logger_middleware.rb [hunk 2]
```diff
@@ -37,15 +37,18 @@ def deep_obfuscate_values(hash)
           if value.is_a?(Hash)
             deep_obfuscate_values(value)
           elsif value.is_a?(String)
-            hash[key] = LOGGABLE_PARAMS.include?(key.to_s) ? value : ('*' * value.length)
+            hash[key] = LOGGABLE_PARAMS.include?(key.to_s) ? value : obfuscate_string(value)
           elsif value.is_a?(Array)
-            hash[key] = value.join(',')
+            hash[key] = value.map { |entry| obfuscate_string(entry) }.join(",")
```
Comment (amiedes) [inline lib/carto/common/rack_logger_middleware.rb]: Just for you to know, in Ruby you can also summarise:

```ruby
value.map { |entry| obfuscate_string(entry) }.join(",")
```

as:

```ruby
value.map(&:obfuscate_string).join(",")
```

But no need to change it 🙂

**Review by amiedes — APPROVED** [review]
Nice work, thanks for taking care of this 🙏

**Pull request merged**